tests/tests/cpld_control_gfsv17

```diff
@@ -78,6 +78,9 @@ export DNATS=0
 export RESTART_N=3
 export RESTART_INTERVAL="${RESTART_N} -1"
 export OUTPUT_FH='0 21 24'
+#CICE forecast day output stream
```

Comment:
Would you please run the tests for a couple of days and show the time stamp change with this update?
Also could you please make the changes to other gfsv17 tests too? Thanks

Reply:
Done. On hera, please see 
`ls /scratch1/NCEPDEV/nems/Nick.Szapiro/tasks/ice_24hrOutput/ufs-weather-model/tests/run_dir/cpld*/history/`

For cpld_control_gfsv17, I ran FHMAX=72 and history includes `iceh.2021-03-23-21600.nc iceh.2021-03-24-21600.nc iceh.2021-03-25-21600.nc` , with time_bounds correct in each

cpld_restart_gfsv17 and cpld_debug_gfsv17 do not have this as they run less than 1 day